### DISCUSSION TITLE
Export TheHive Cases to New Server

### DISCUSSION THREAD

**Discussion by molfar101** [Security-Onion-Solutions/securityonion#8278] [Unsupported Versions] [answered]
Hello,

I'm considering to update SO to the newest version but the problem is that it doesn't support TheHive anymore. SO recommends to migrate to an external instance of TheHive. However, I need to keep all the cases and migrate them to a new instance. I didn't find any documentation how to do it. I would appreciate any ideas how to accomplish this task.

Thank you

**Comment by dougburks** [comment, marked as answer]
As you mentioned, we don't support TheHive anymore. If you want to continue to use TheHive, then you should reach out to TheHive community for support.
